pandas-stubs/core/indexes/multi.pyi

```diff
@@ -92,7 +92,6 @@ class MultiIndex(Index):
     def to_flat_index(self): ...
     @property
     def is_all_dates(self) -> bool: ...
-    def is_lexsorted(self) -> bool: ...
     def lexsort_depth(self): ...
```

Comment:
`lexsort_depth()` was deprecated in 1.3, so we should remove that

Reply:
Removed.